addons/wowl/static/tests/helpers.ts

```diff
@@ -133,21 +135,23 @@ export function click(el: HTMLElement, selector?: string) {
 export function makeFakeUserService(fullContext: boolean = false): typeof userService {
   return {
     name: "user",
-    deploy() {
+    deploy(env: OdooEnv, odoo: Odoo): UserService {
       const context = fullContext
         ? { lang: "en_us", tz: "Europe/Brussels", uid: 2, allowed_company_ids: [1] }
         : ({ uid: 2 } as any);
-      return {
+
+      const res = Object.assign({}, localizationParameter, {
         context,
         userId: 2,
         userName: "admin",
         isAdmin: true,
         partnerId: 3,
-        allowed_companies: [[1, "YourCompany"]],
-        current_company: [1, "YourCompany"],
+        allowed_companies: [[1, "YourCompany"]] as UserCompany[],
```

Comment:
infered type for [[1, "YourCompany"]] is (number|string)[][]. We have to specify this